src/main/java/com/google/opensesame/projects/ProjectDatastore.java

```diff
@@ -19,8 +24,17 @@
    * @return Returns the key associated with the datastore entity.
    */
   public static Key addProjectEntity(ProjectEntity projectEntity) {
+    if (projectEntity.getNumMentors() <= 0) {
+      System.out.println(
+          "Saving project ("
+              + projectEntity.repositoryId
+              + ") with zero mentors."
+              + " This is most likely a mistake, as projects should only exist in the"
+              + " datastore with one or more mentors.");
+    }
```

Comment:
Is this intended to go to master?

Reply:
Yes, unless you think it's not worth including. We designed our backend with the assumption that all projects would have at least one mentor, so I wanted to include a soft check for that assumption here.